ibm/config.go

```diff
@@ -2066,6 +2076,19 @@ func (c *Config) ClientSession() (interface{}, error) {
 	}
 	session.resourceManagerAPI = resourceManagerClient
 
+	ibmCloudShellClientOptions := &ibmcloudshellv1.IBMCloudShellV1Options{
```

Comment:
Can you please provide a environmental variable to set to stage environmental and support for private(if supported)...please see other client intialization
```
apicurl := contructEndpoint(fmt.Sprintf("api.%s.apigw", c.Region), fmt.Sprintf("%s/controller", cloudEndpoint))
	if c.Visibility == "private" || c.Visibility == "public-and-private" {
		apicurl = contructEndpoint(fmt.Sprintf("api.private.%s.apigw", c.Region), fmt.Sprintf("%s/controller", cloudEndpoint))
	}
	APIGatewayControllerAPIV1Options := &apigateway.ApiGatewayControllerApiV1Options{
		URL:           envFallBack([]string{"IBMCLOUD_API_GATEWAY_ENDPOINT"}, apicurl),
		Authenticator: &core.NoAuthAuthenticator{},
	}
	apigatewayAPI, err := apigateway.NewApiGatewayControllerApiV1(APIGatewayControllerAPIV1Options)
	if err != nil {
		session.apigatewayErr = fmt.Errorf("Error occured while configuring  APIGateway service: %q", err)
	}
	session.apigatewayAPI = apigatewayAPI
	```

Reply:
we do not currently support private endpoints

Reply:
done